perun-core/src/main/java/cz/metacentrum/perun/core/entry/GroupsManagerEntry.java

```diff
@@ -1361,6 +1411,7 @@ public void synchronizeGroupsStructures(PerunSession sess) throws InternalErrorE
 		// Authorization
 		if (!AuthzResolver.isAuthorized(sess, Role.VOADMIN, parentGroup)
 		        && !AuthzResolver.isAuthorized(sess, Role.VOOBSERVER, parentGroup)
+				&& !AuthzResolver.isAuthorized(sess, Role.PERUNOBSERVER)
```

Comment:
You should use the same indentation, this looks weird (for all methods with the same problem). There are spaces in original, but tabs in your commit. You can use both, but please all the same.